src/common/CMakeLists.txt

```diff
@@ -1,8 +1,22 @@
+
+set (MONGOC_ENABLE_DEBUG_ASSERTIONS 0)
+
+if (ENABLE_TEST_ASSERTIONS)
```

Comment:
`ENABLE_DEBUG_ASSERTIONS`

Reply:
That one's correct I think? config.h.in will set the macro `MONGOC_ENABLE_DEBUG_ASSERTIONS` to whatever the variable cmake variable `MONGOC_ENABLE_DEBUG_ASSERTIONS` equals.

Reply:
Oh sorry nevermind.